src/components/Dropdown/Dropdown.tsx

```diff
@@ -518,10 +541,10 @@ const Dropdown = ({
           {options.map(option => (
             <StyledOptionItem
               id={`${name}-option-${option.id}`}
+              data-test-id={`${internalId.current}-option-${option.id}`}
               key={`${name}-option-${option.id}`}
-              onBlur={handleBlur}
               onClick={() => handleSelect(option.id)}
-              tabIndex={tabIndex}
+              tabIndex={-1}
```

Comment:
Just want to make sure this still works with tabbing to the dropdown after this change, (can you confirm that for me or send me to a place I can test for myself)?

Reply:
You can tab TO the dropdown, which opens it. Then the arrow keys control the options list, and Enter selects/deselects the options. Pressing Tab again closes the dropdown and moves to the next input/tabIndex on the page.

Reply:
This was the original problem the PR was made to fix. Before, tab wouldn't close the dropdown, and you'd have to go through all the options using tab before you could leave.